Dockerfile

```diff
@@ -1,5 +1,5 @@
 # syntax=docker/dockerfile:1.4
-FROM node:18.19.0-alpine3.18 AS base
+FROM node:20.11.0-alpine3.19 AS base
```

Comment:
should we use 20-alpine instead?

Reply:
I would recommend specific versions as we can be less prone to future vulnerabilities and have more control.

We can frequently upgrade this dependency along with the others like NPM packages.

Reply:
we need to plan node upgrades more often than